flink-connectors/flink-hbase/src/main/java/org/apache/flink/table/descriptors/HBase.java

```diff
@@ -0,0 +1,118 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.descriptors;
+
+import java.util.Map;
+
+import static org.apache.flink.table.descriptors.ConnectorDescriptorValidator.CONNECTOR_VERSION;
+import static org.apache.flink.table.descriptors.HBaseValidator.CONNECTOR_TABLE_NAME;
+import static org.apache.flink.table.descriptors.HBaseValidator.CONNECTOR_TYPE_VALUE_HBASE;
+import static org.apache.flink.table.descriptors.HBaseValidator.CONNECTOR_WRITE_BUFFER_FLUSH_INTERVAL;
+import static org.apache.flink.table.descriptors.HBaseValidator.CONNECTOR_WRITE_BUFFER_FLUSH_MAX_ROWS;
+import static org.apache.flink.table.descriptors.HBaseValidator.CONNECTOR_WRITE_BUFFER_FLUSH_MAX_SIZE;
+import static org.apache.flink.table.descriptors.HBaseValidator.CONNECTOR_ZK_NODE_PARENT;
+import static org.apache.flink.table.descriptors.HBaseValidator.CONNECTOR_ZK_QUORUM;
+
+/**
+ * Connector descriptor for Apache HBase.
+ */
+public class HBase extends ConnectorDescriptor {
+	private DescriptorProperties properties = new DescriptorProperties();
+
+	public HBase() {
+		super(CONNECTOR_TYPE_VALUE_HBASE, 1, true);
+	}
+
+	/**
+	 * Set the Apache HBase version to be used. Optional.
+	 *
+	 * @param version HBase version. E.g., "1.4.3".
+	 */
+	public HBase version(String version) {
+		properties.putString(CONNECTOR_VERSION, version);
+		return this;
+	}
+
+	/**
+	 * Set the HBase table name, Required.
+	 *
+	 * @param tableName Name of HBase table.  E.g., "testNamespace:testTable", "testDefaultTable"
+	 */
+	public HBase tableName(String tableName) {
+		properties.putString(CONNECTOR_TABLE_NAME, tableName);
+		return this;
+	}
+
+	/**
+	 * Set the zookeeper quorum address to connect the HBase cluster. Required.
+	 *
+	 * @param zookeeperQuorum zookeeper quorum address to connect the HBase cluster. E.g., "localhost:2181,localhost:2182,localhost:2183".
+	 */
+	public HBase zookeeperQuorum(String zookeeperQuorum) {
+		properties.putString(CONNECTOR_ZK_QUORUM, zookeeperQuorum);
+		return this;
+	}
+
+	/**
+	 * Set the zookeeper node parent path of HBase cluster. Optional.
+	 *
+	 * @param zookeeperNodeParent zookeeper node path of hbase cluster. E.g, "/hbase/example-root-znode".
+	 */
+	public HBase zookeeperNodeParent(String zookeeperNodeParent) {
+		properties.putString(CONNECTOR_ZK_NODE_PARENT, zookeeperNodeParent);
+		return this;
+	}
+
+	/**
+	 * Set threshold when to flush buffered request based on the memory byte size of rows currently added . Default to <code>2mb</code>. Optional.
+	 *
+	 * @param writeBufferFlushMaxSize threshold (Byte size) to flush a buffered request. E.g, 2097152 (2MB).
+	 */
+	public HBase writeBufferFlushMaxSize(long writeBufferFlushMaxSize) {
```

Comment:
How about using `String` as the parameter, so that users can set "10mb". This aligns with the SQL DDL and `Elasticsearch#bulkFlushMaxSize` accepts a String format memory in the same way.

Reply:
OK, let me have a check.